definitions/mlops-machine_learning_training/definition.yml

```diff
@@ -0,0 +1,19 @@
+domain: MLOPS
+type: MACHINE_LEARNING_TRAINING
```

Comment:
Would TRAINING_JOB be more accurate?

Reply:
I want to leave it more general (although might change in the future)

Reply:
Changing it in the future will mean breaking all entities and creating new ones.
With their alerts, metrics, favorites etc.

If we think in the future should be more specific better do it now.

Reply:
ok, changed, thanks